spec/helpers/argo_helper_spec.rb

```diff
@@ -14,12 +14,13 @@
       allow(@object).to receive(:can_manage_item?).and_return(true)
       allow(@object).to receive(:pid).and_return(@item_id)
       desc_md = double(Dor::DescMetadataDS)
-      id_md   = double(Dor::DescMetadataDS)
+      @id_md  = double(Dor::IdentityMetadataDS)
```

Comment:
arguably, this switch to `Dor::IdentityMetadataDS` could've been it's own commit, since it's fixing an incorrect mocking that wasn't specific to the test i added, but it seemed fine to bundle this in since it's so small.  happy to break out if that's preferred.  making `id_md` an instance var was specific to the new test, which needed to `allow(@id_md).to receive(:otherId).with('catkey').and_return(['1234567'])`.